ibm/ibm-how-to/launch-ubuntu-instances.rst

```diff
@@ -0,0 +1,153 @@
+Launch a Virtual Server Instance on IBM Cloud
+=============================================
+
+This tutorial guides you through the process of launching a Virtual Server Instance (VSI) on IBM Cloud using Ubuntu. Ensure you have an active IBM Cloud account before starting.
```

Comment:
How strict are we adhering to diataxis with these? You want to refer to this document as a 'how-to' instead of a 'tutorial'.

Reply:
That is a great question...

Reply:
Yes, this is not a tutorial but a how-to.